docs/webserver.rst

```diff
@@ -89,6 +89,35 @@ To use files that already exist on the controlled server, use:
           key: /etc/ssl/private/ssl-cert-snakeoil.key
           crt: /etc/ssl/certs/ssl-cert-snakeoil.pem
 
+Alternatively, you can use certbot to create and renew certificates.
+Certificates are in the usual ``/etc/letsencrypt/live/HOST_NAME`` folders.
+If you specify a global ``certbot_hosts`` list variable, then certificates managed by certbot from Let's Encrypt will be used for all matching hosts.
+
+.. code-block:: yaml
+
+   certbot_hosts:
+     - one.mcsmith.org
+     - two.mcsmith.org
+
+Or if you have the ``inventory_hostname`` variable defined:
+
+.. code-block:: yaml
+
+   certbot_hosts:
+     - "{{ inventory_hostname }}"
+
+To automatically renew the certificates, you need to set the time you want to attempt renewal on a daily basis with the ``certbot_renew_at`` global variable.
+
+.. code-block:: yaml
+
+    certbot_renew_at:
+      minute: 30
+      hour: 3
+
```

Comment:
If python3-certbot-nginx does in fact create a cron job, then this section is not necessary. I found no such cron job on my server, so this was necessary for me. Maybe I'm doing something wrong?